addons/advanced_fatigue/stringtable.xml

```diff
@@ -113,6 +113,7 @@
         </Key>
         <Key ID="STR_ACE_Advanced_Fatigue_SwayFactor">
             <English>Sway factor</English>
+            <German>Stabilisierungsfaktor</German>
```

Comment:
Sway == Pendeln/Schwanken.
Pendelfaktor ?
"Stabilisierungsfaktor" implies that higher value == more stable. But it's the opposite.

Reply:
Verwacklungsfaktor

Reply:
That sounds so funny tho :D
But I think it's best match.